shared/actions/tracker.js

```diff
@@ -51,14 +51,13 @@ export function stopTimer (): Action {
 }
 
 export function registerTrackerChangeListener (): TrackerActionCreator {
-  return dispatch => {
+  return (dispatch, getState) => {
     const params = {
       'keybase.1.NotifyTracking.trackingChanged': ({username}) => {
-        dispatch({
-          type: Constants.userUpdated,
-          payload: {},
-        })
-        dispatch(getProfile(username))
+        const trackerState = getState().tracker.trackers[username]
+        if (trackerState && trackerState.type === 'tracker' && !trackerState.closed) {
```

Comment:
we have to get rid of the closed check.

This is the only code path that updates the tracker info based on a notification. We need this to update visible profiles or search panels. This flow shouldn't do anything to stop those from getting updated so just the check for tracker i think is fine